ProjectLighthouse.Servers.Website/Pages/Login/BannedUserPage.cshtml

```diff
@@ -30,6 +32,12 @@
                 }
             </span>    
         }
+        else
+        {
+            <span>
+                @Model.Translate(ModerationStrings.SuspensionExpiration, Model.Translate(GeneralStrings.Unknown))
+            </span>
+        }
 
         <ul>
```

Comment:
According to HTML specs and Rider analysis, you shouldn't be putting a list inside a paragraph object, you can either move the list outside the paragraph tag or turn it into a div